docs/customization.rst

```diff
@@ -118,6 +123,34 @@ Unlike, newer versions of mkdocs-material theme, this theme also supports the us
 
         :hero: Configuration options to personalize your site.
 
+.. confval:: hide-navigation
+
+    This theme uses the ``:hide-navigation:`` metadata to hide the page's navigation menu (on
+    large enough viewports only).
```

Comment:
If specified, hides the global navigation sidebar shown on the left side of the page.  By default, the navigation menu is shown if the browser viewport is sufficiently wide.

Reply:
"left" is relative to the `direction` setting.

Reply:
Oh, I wasn't aware of that.  I guess we could say "left (or right in rtl langauge mode)" or leave out the side altogether.  The main thing was that I wanted to clarify that it was the global toc rather than the local toc, as users may not know the distinction between nav vs toc.